internal/network/command_private_link_attachment_connection_create.go

```diff
@@ -19,21 +19,22 @@ func (c *command) newPrivateLinkAttachmentConnectionCreateCommand() *cobra.Comma
 		RunE:  c.privateLinkAttachmentConnectionCreate,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: "Create a named AWS PrivateLink attachment connection.",
+				Text: "Create a named PrivateLink attachment connection.",
```

Comment:
Good point, I think it would be one of 
- Private Link
- PrivateLink

Just not sure if the standard is to have the space or not, I'll follow up in my team

Reply:
I think AWS uses PrivateLink and Azure uses Private Link.

Reply:
I think we use both "Private Link" and "Private Link` in our doc 🤔

Reply:
Team decided that it should be "Private Link", made the changes accordingly in the files!